ocw_oer_export/create_csv.py

```diff
@@ -0,0 +1,172 @@
+"""
+Module for creating OER-template CSV file with data extracted from MIT OpenCourseWare API.
+"""
+import csv
+import os
+import logging
+
+
+from .client import extract_data_from_api
+from .data_handler import extract_data_from_json
+from .constants import API_URL
+from .utilities import text_cleanup
+
+
+def create_ocw_topic_to_oer_subject_mapping(path=None, file_name=None):
+    """Returns a mapping (dictionary) of OCW topics to OER subjects."""
+    if path is None:
+        path = os.path.dirname(__file__)
+
+    if file_name is None:
+        file_name = "mapping_files/ocw_topic_to_oer_subject.csv"
+
+    file_path = os.path.join(path, file_name)
+    with open(file_path, newline="", encoding="utf-8") as csvfile:
+        reader = csv.DictReader(csvfile)
+        return {row["OCW Topic"]: row["OER Subject"] for row in reader}
+
+
+def get_cr_subjects(ocw_topics_mapping, ocw_course_topics):
+    """Get Course Resource Subjects based on OCW topics."""
```

Comment:
**Suggestion:** I think it might be worth expanding docstrings slightly. For example:
```
    """
    Get OER formatted Course Resource Subjects list.
    
    Since distinct OCW topics can map to the same OER subject, duplicate subject
    values are omitted.
    """
```

Readers will probably realize fairly quickly (since its the natural of this package) but I would write the docstrings to be clear that the func is transforming from OCW values to OER values.